CONTRIBUTING-TYPESCRIPT.md

```diff
@@ -16,18 +16,18 @@ This guide covers TypeScript-specific setup and development for AgentKit.
 
 ## Development Setup
 
-AgentKit uses Node.js v18.x or higher and npm 8.x or higher.
+AgentKit uses Node.js v18.x or higher and pnpm 10.7.x or higher.
 
-You can run the following commands in your terminal to check your local Node.js and npm versions:
+You can run the following commands in your terminal to check your local Node.js and pnpm versions:
 
 ```bash
 node --version
-npm --version
+pnpm --version
 ```
 
-If the versions are not correct or you don't have Node.js or npm installed, download through [nvm](https://github.com/nvm-sh/nvm).
+If the versions are not correct or you don't have Node.js or pnpm installed, download Node.Js through [nvm](https://github.com/nvm-sh/nvm) and then install [pnpm](https://pnpm.io/installation).
```

Comment:
While we're at it, let's recommend devs use [fnm](https://github.com/Schniz/fnm) instead of `nvm`. It's faster